ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2600,6 +2622,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy-minimal-permission
+- as: nutanix-ipi-sdn-live-migration-ovn-f28
+  cluster: build01
+  cron: 6 19 6 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
```

Comment:
We have FIPS_ENABLED here, but not indicated in the test name.  So I'm not sure whether you want to test fips or not.

Reply:
ah, thanks for catching it. 
I'd like to enable it even though it's not important for migration.  Update the profile name with `fips`, thanks